war/pom.xml

```diff
@@ -783,7 +784,8 @@ THE SOFTWARE.
                   <goal>yarn</goal>
                 </goals>
                 <configuration>
-                  <arguments>lint</arguments>
+                  <arguments>lint:ci</arguments>
+                  <skip>${yarn.lint.skip}</skip>
```

Comment:
extracted from https://github.com/jenkinsci/jenkins/pull/6863/

figured may as well given touching the same area